src/components/pages/ResearcherPage.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import Helmet from 'react-helmet'
+import { ethers } from 'ethers'
+import { Link } from 'react-router-dom'
+import { createCanvas } from 'canvas'
+import { renderIcon } from '@download/blockies'
+import { Query } from 'react-apollo'
+import { ResearcherNameAndAddress } from '~/components/researchers/ResearcherNameAndAddress'
+import { HeroBetaCallout } from '~/components/HeroBetaCallout'
+import { FooterContainer } from '~/components/layout/Footer'
+import { ErrorMessage } from '~/components/ErrorMessage'
+import { PageDetailsLoader } from '~/components/PageDetailsLoader'
+import { PackageItemListRow } from '~/components/PackageItemListRow'
+import { ScrollToTop } from '~/components/ScrollToTop'
+import { projectResearcherVouchedPackages } from '~/projections/projectResearcherVouchedPackages'
+import { vouchingQueries } from '~/queries/vouchingQueries'
+import { displayWeiToEther } from '~/utils/displayWeiToEther'
+import * as routes from '~/../config/routes'
+
+export class ResearcherPage extends PureComponent {
+  static propTypes = {
+    match: PropTypes.object.isRequired
+  }
+
+  render () {
+    const address = this.props.match.params.address
+    const canvas = createCanvas(50, 50)
+    const icon = renderIcon(
+      {
+        seed: address,
+        color: '#dfe',
+        bgcolor: '#fff',
+        size: 10, // width/height of the icon in blocks, default: 10
+        scale: 5 // width/height of each block in pixels, default: 5
+      },
+      canvas
+    )
```

Comment:
Let's make sure to confirm a generated icon is appropriate during the meeting tomorrow.  There is a chance we could pull it from one of the Challenge URIs.